### DISCUSSION TITLE
Print my printscreen

### DISCUSSION THREAD

**Discussion by staman81** [flameshot-org/flameshot#3490] [Q&A]
I know, this has been requested many times since 2020 but will this ever be implemented? Most of the request has been closed without anything happen in Flameshot

We would love to have a option/button to print the selection done via flameshot (like Lightshot does)

Thank you

**Comment by mmahmoudian** [comment]
No it will never happen in Flameshot. As have been mentioned in those open and closed issues, this is something that is listed to be implemented via the Plugins system.

If you are on Linux, for now your best way is to use our CLI. If you are on Windows, you should wait until this is implemented.

If you really miss such feature, please consider putting code bounty on the issues you like to be implemented. More information and explanation is available here:

https://flameshot.org/donate/

**Reply by staman81** [reply]
I'm a Windows admin and waiting patiently since 2020 :) 
Just wonder ..

Thank you for your explanation
